Packs/CommonScripts/Scripts/ShowLocationOnMap/ShowLocationOnMap_test.py

```diff
@@ -63,3 +63,29 @@ def test_invalid_loc(loc, mocker):
         del (sys.modules['ShowLocationOnMap'])
     with pytest.raises(Exception, match='return_error_called'):
         import ShowLocationOnMap  # noqa: F401
+
+
+INVALID_LOC_TYPE = [None, {}, []]
+
+
+@pytest.mark.parametrize('loc', INVALID_LOC_TYPE)
+def test_not_string_indicator_value(mocker, loc):
+    """
+    Given:
+    - Demisto args with invalid types as indicator value.
+
+    When:
+    - Trying to parse it into lat and lng separately.
+
+    Then:
+    - Ensure that either return error or return results is not raised and nothing returns from the script.
```

Comment:
Actually, that might be a bit problematic. We need the script to return something (this is why I have added the else)

Reply:
very well ill add that else statement as well